core-primitives/ocall-api/src/lib.rs

```diff
@@ -91,11 +91,17 @@ pub trait EnclaveOnChainOCallApi: Clone + Send + Sync {
 		header: &H,
 	) -> Result<StorageEntryVerified<V>>;
 
-	fn get_multiple_storages_verified<H: Header<Hash = H256>, V: Decode>(
+	fn get_multiple_storages_verified_and_decoded<H: Header<Hash = H256>, V: Decode>(
 		&self,
 		storage_hashes: Vec<Vec<u8>>,
```

Comment:
Furthermore, when updating the storage ( https://github.com/encointer/community-sidechain/blob/349f7cb5623b1083a382b770e56d0b920af2fa5f/core-primitives/stf-executor/src/executor.rs#L120), I don't think we need the decoding. 
This would also be the case at https://github.com/encointer/community-sidechain/blob/349f7cb5623b1083a382b770e56d0b920af2fa5f/core-primitives/stf-executor/src/executor.rs#L180 and at https://github.com/encointer/community-sidechain/blob/349f7cb5623b1083a382b770e56d0b920af2fa5f/core-primitives/stf-executor/src/executor.rs#L210.
If you agree with this fix, I will replace them with `get_multiple_storages_verified` and I will check all other uses

Reply:
I think this makes sense 👍 This is worth an upstream PR!